dist/select/ds4/select.css

```diff
@@ -42,6 +42,7 @@ span.select {
 .select--large select {
```

Comment:
Is the fix only need for the large size?

Reply:
I didnt see a problem with alignment on normal sizes. So put it only for large.